src/vtkIECTransformLogic.h

```diff
@@ -157,7 +157,27 @@ class VTK_IEC_TRANSFORM_LOGIC_EXPORT vtkIECTransformLogic : public vtkObject
   {
     return this->IECTransforms;
   }
+  
```

Comment:
Cross-check if we need include <cstdint> or <array> (to avoid transitive includes)